newlib/libc/sys/vita/include/netinet/in.h

```diff
@@ -34,8 +34,8 @@
 #include <psp2/net/net.h>
 
 #define IPPROTO_IP		SCE_NET_IPPROTO_IP
-#define IPPROTO_ICMP	SCE_NET_IPPROTO_ICMP
-#define IPPROTO_IGMP	SCE_NET_IPPROTO_IGMP
+#define IPPROTO_ICMP		SCE_NET_IPPROTO_ICMP
+#define IPPROTO_IGMP		SCE_NET_IPPROTO_IGMP
```

Comment:
this file was mixed ts 4 and 8.
but after that only remain `struct sockaddr_in` part.
can you fix this?